includes/admin/load.php

```diff
@@ -330,8 +330,27 @@ static function ( $name ) {
 	}
 
 	if ( isset( $_GET['activate'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		$message = __( 'Feature activated.', 'performance-lab' );
+
+		$plugin_settings_link = perflab_get_plugin_settings_link( $_GET['activate'] ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+
+		if ( $plugin_settings_link ) {
+			$p = new WP_HTML_Tag_Processor( $plugin_settings_link );
+			if ( ! empty( $p->next_tag( array( 'tag_name' => 'A' ) ) ) && ! empty( $p->get_attribute( 'href' ) ) && ! is_bool( $p->get_attribute( 'href' ) ) ) {
+				/* translators: %s is the settings URL */
+				$message .= ' ' . sprintf( __( 'Review <a href="%s">settings</a>.', 'performance-lab' ), esc_url( $p->get_attribute( 'href' ) ) );
```

Comment:
I think this this would be a bit simpler for the necessary type checks:

```suggestion
			if ( ! empty( $p->next_tag( array( 'tag_name' => 'A' ) ) ) && $p->get_attribute( 'href' ) ) {
				/* translators: %s is the settings URL */
				$message .= ' ' . sprintf( __( 'Review <a href="%s">settings</a>.', 'performance-lab' ), esc_url( (string) $p->get_attribute( 'href' ) ) );
```